src/window.rs

```diff
@@ -780,6 +780,18 @@ impl Window {
     pub fn drag_window(&self) -> Result<(), ExternalError> {
         self.window.drag_window()
     }
+
+    /// Modifies whether the window catches cursor events.
+    ///
+    /// If `false`, this will catch the cursor events. If `true`, this will ignore the cursor.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **iOS / Android / Web / X11 / Wayland:** Unsupported.
+    #[inline]
+    pub fn set_ignore_cursor_events(&self, ignore: bool) -> Result<(), ExternalError> {
```

Comment:
A bit of bikeshedding but I would prefer some affirmative variant of the method name as `set_ignore_cursor_events(false)` might be hard to read. My suggestion would be `set_cursor_hittest`.